internal/pkg/cli/storage_init.go

```diff
@@ -281,8 +281,12 @@ func (o *initStorageOpts) Ask() error {
 	if wkld.Type == manifest.RequestDrivenWebServiceType && o.storageType == rdsStorageType {
 		log.Warningf(`%s storage is launched in private subnets of your environment's VPC. 
 To reach the database from your %s please enable the HTTP Data API: 
-https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html
-`, rdsStorageTypeOption, manifest.RequestDrivenWebServiceType)
+https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html. 
```

Comment:
You know what would be extra cool 🤩 , is adding a "Show and Tell" post that showcases how to do this.

1. Create a sample repo in Copilot on GitHub with exactly  this setup.
   Maybe we can still Massimo's code: https://github.com/aws-containers/votingapp and update it to read the db secret from SecretsManager, and use Aurora as a DB instead.
2. Then walk through the commands on how to deploy it.
3. Explain what's happening under the hood.

Reply:
sounds good!!